server/routes/notifications.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const {
+    createNotification,
+    readNotification,
+    getAllNotifications,
+    getUnreadNotifications
+} = require('../controllers/notification');
+
+router.route('/create').post(createNotification);
+
+router.route('/read').post(readNotification);
```

Comment:
You could make this dynamic instead of passing the id in the request body.
`router.route('/read/:id').post(readNotification);`